pkgs/top-level/aliases.nix

```diff
@@ -151,6 +151,7 @@ mapAliases ({
   cadence = throw "cadence has been removed from nixpkgs, as it was archived upstream"; # Added 2023-10-28
   cask = emacs.pkgs.cask; # Added 2022-11-12
   canonicalize-jars-hook = stripJavaArchivesHook; # Added 2024-03-17
+  cargo-deps = throw "cargo-deps has been removed as the repository is deleted"; # Added 2024-04-09
```

Comment:
Should we mention the [alternatives](https://crates.io/crates/cargo-deps#alternatives) in the readme?

Reply:
Sorry, which readme?